src/templates/pycontw-2020/contents/en/about/privacy_policy.html

```diff
@@ -0,0 +1,127 @@
+	its Code 001 Type for identifying individuals.
+</p>
+
+<h4>Your Rights of Personal Information</h4>
+
+<p>
+	You could exercise your rights (which would be mentioned below) of the personal information by writing to us (either
+	by paper or by electronic means)
+</p>
+
+<ol class="custom_ol">
+	<li>Any inquiries or requests for viewing the personal information</li>
+	<li>Any requests to make duplications of the personal information</li>
+	<li>Any requests to add supplements or correct the personal information</li>
+	<li>Any requests to discontinue the collection, processing or using of personal information</li>
+	<li>Any requests to delete the personal information unless you do not agree the process of registration and legal
+		regulation
+	</li>
+</ol>
+<p>
+	We would keep it and change your request all according to Taiwan’s legal obligations.
+	If you would like to have your personal information modified or updated, you could do it on the registration page or
+	email us.
+</p>
+<h4>Cancelling Your Participating and Deleting Your Personal Information</h4>
+
+<p>
+	You may request us to cease the use of your personal information by notifying us through <a
+	href="mailto:organizers@pycon.tw">mail</a> and we shall refrain from using your personal information from then
+	onwards. You are free to provide as much or as little information as you like. However, in the case that the provided
+	information is erroneous, insufficient, or that you request us to cease the usage of your data, then for the proper
+	operation of the event, we may reject your registration and participation.
+</p>
+
+<h4>Questions for Privacy Protection Policy</h4>
+<p>
+	If you have any questions regarding the gathering and usage of your personal information, please feel free to contact
+	us at <a href="mailto:organizers@pycon.tw">organizers@pycon.tw</a>.
+</p>
+
+<h4>Privacy: Collecting Personal Information during COVID-19 Outbreak</h4>
+<p>
+	The purpose of collecting personal information in accordance with the Personal Data Protection Law and epidemic
+	prevention and control requirements is to best respond and ensure health security for all attendees in COVID-19
+	outbreak. It is necessary to collect, manage, and use relevant personal information. Please read below for
+	details.
+</p>
+<p>
+	PyCon TW 2020 collects personal data of attendees for the purpose of contact tracing during the outbreak of COVID-19.
+	All collected information will be deleted after 28 days. attendees’ access in each venue will be recorded for epidemic
+	control measures. We will inform everyone that if any suspected COVID-19 and provide such information for CECC for the
+	purpose of preventing spread of COVID-19 and tracing contacts.
+</p>
+
+<h4>License (CC BY-SA 3.0 TW)</h4>
+<p>This document is licensed under a <a target="_blank" rel="noopener" href="https://creativecommons.org/licenses/by-sa/3.0/tw/deed.zh_TW">Attribution-ShareAlike
```

Comment:
I'll approve this PR first, but after this, I'd suggest change this link into the English version https://creativecommons.org/licenses/by-sa/3.0/tw/deed.en

Sorry I didn't notice this last time.

Reply:
ok~